src/components/GitHub/GitHubProjects.vue

```diff
@@ -241,4 +239,26 @@ export default {
 .margin {
   margin-top: 60px;
 }
+#addButton {
+  align-self: center;
+  margin: 0.5em 0;
+  border-radius: 2em;
+  padding: 0.5em 2em;
+  cursor: pointer;
+  background-color: #86B1B1;
+  border-width: 0;
+  font-weight: bold;
+}
+
+#closeButton {
```

Comment:
The code for this id is basically the same of the id above it. If you want the same style in several places, create a css class and added it, instead of Ids.